### PR TITLE
Getting started card completed

### LINKED ISSUE #2562: [Dashboard v1] Getting started card on has no content
## About
Getting Started Card | Problem
--- | ---
image | There is no content available for these, and currently this card is hidden. Any relevant content would help, since this card will otherwise never show and be tested.

_**Note:** I see this card being used for navigation and actions done within the dashboard or view. Is this a correct assumption_

/assign @sarahmaddox 
/assign @abhi-g 
/assign @ajayalfred 
/assign @avdaredevil 

/cc @jlewi

### PR DESCRIPTION
**Pull request by avdaredevil**
## About
Getting started card completed

Card in normal View | Responsive View
--- | ---
image | image

#### Fixes #2562

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2651)

### CODE REVIEW

File: components/centraldashboard/webpack.config.js [hunk 1]
```diff
@@ -81,7 +81,7 @@ module.exports = {
                     }
                 }
             },
-            {
+            (process.platform == 'win32'?{}:{
```
Comment (prodonjs) [inline components/centraldashboard/webpack.config.js]: The Windows issues seem like they may be localized to your environment. I set up a Windows 2016 server on GCE, checked from master, and we able to get both the `dev` and `build` npm commands to work as expected without any changes to the eslint settings.

The site doesn't work in IE11 (per our Babel settings), but it worked perfectly fine in Chrome on Windows.

File: components/centraldashboard/public/components/main-page.js [hunk 2]
```diff
@@ -54,11 +54,32 @@ export class MainPage extends PolymerElement {
...
                 value: [
-                    {text: 'Build a model in a notebook', icon: 'donut-large'},
-                    {text: 'Launch a pipeline', icon: 'donut-large'},
-                    {text: 'Deploy a solution', icon: 'donut-large'},
-                    {text: 'Compare run output', icon: 'donut-large'},
-                    {text: 'Share solution on AI Hub', icon: 'donut-large'},
+                    {
+                        text: 'Getting started with Kubeflow',
+                        desc: 'Quickly get running with your ML workflow on an existing Kubernetes installation',
+                        link: 'https://www.kubeflow.org/docs/started/getting-started/',
```
Comment (prodonjs) [inline components/centraldashboard/public/components/main-page.js]: It seems a bit odd that these links open externally versus using the iframe view which would be activated if the user clicked Kubeflow Docs from the sidenav. My open PR #2614 will refactor this section and add the router, but I'm curious if you think we should make these links appear in the iframe as well.

Comment (avdaredevil) [reply components/centraldashboard/public/components/main-page.js]: Your logic makes perfect sense, and that was my initial goal. However, that is no longer the desired behavior as mentioned in #2546. 

I am waiting for your PR to then refactor all `link`s to `href`s

File: components/centraldashboard/webpack.config.js (hunk 1 above)
Comment (avdaredevil) [reply components/centraldashboard/webpack.config.js]: Ok I'll work on this a little more to get it working. Seems really wierd

**Comment by avdaredevil** [comment]
Correct me if I am wrong, but I18n isn't on the Agenda for landing page right? @ajayalfred

**Comment by prodonjs** [comment]
I still don't understand why your specific environment is not letting us use the [standard method](https://eslint.org/docs/user-guide/configuring#using-configuration-files-1) of putting the configuration file in the directory where it will be applied. I don't want to hold things up though so we'll have to investigate that later.

/lgtm

**Pull request merged**